invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/components/MetadataTable/MetadataTable.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import { Table } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+export default class MetadataTable extends Component {
+  _renderRows() {
+    const labelWidth = this.props.labelWidth;
+    const dataColWidth = this.props.dataColumnWidth;
+    const entry = this.props.entry;
+
+    return this.props.columns.map(column => (
```

Comment:
I am not sure we want to have this generic component... I have the feeling we will customize this view to look good and it is only for documents that we don't know the fields in advance, so I am not sure we will re-use this.

In any case, these are actually rows and not columns if I understand correctly:
```
rowLabel: rowValue,
rowLabel: rowValue,
rowLabel: rowValue,
```
Would it be more easier to understand if we call it?
```
this.props.rows.map(rows => (
    row.name
    row.value
)
```

Reply:
done!